website_and_docs/content/blog/2024/selenium-vs-blog-posts.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Selenium Vs … blog posts"
+linkTitle: "Selenium Vs … blog posts"
+date: 2024-01-09
+tags: ["selenium"]
+categories: ["general"]
+author: David Burns ([@AutomatedTester](https://twitter.com/automatedtester))
+description: >
+   This blog post discusses the click bait posts out there comparing selenium, cypress, and playwright. How none of these are meaningful or helpful.
+---
+
+The easiest way to clickbait a blog post about automated testing is to compare Selenium against another tool in the space with a catchy title especially when it talks down about the incumbents.
+
+Unfortunately, this can lead to muddying the waters of what features are available in any of the products out there especially when we compare apples to apples.
+
+Selenium has always been a great tool for browser automation. Fortunately for the project, it has become the tool of choice for testing web applications for nearly 2 decades. The area this project has focused on is building out the hard parts of browser automation that are increasingly hard. Stable APIs and scalability of the infrastructure to run Selenium. It hasn’t focused on how people test with it because there are very good test frameworks out there and having to do it for 5 different languages is a non-trivial amount of engineering effort.
```

Comment:
"hard. Stable APIs and scalability of the infrastructure to run Selenium" isn't a sentence